web-app/css/record-view.css

```diff
@@ -1,3 +1,7 @@
+audio {
```

Comment:
Instead of setting a style for the tag, should have made a class for record view's audio player.